etc/supervisor.conf

```diff
@@ -0,0 +1,15 @@
+[program:unicore_mc]
+command = /var/praekelt/python/bin/gunicorn --bind 0.0.0.0:7001 project.wsgi
+stdout_logfile = /var/praekelt/logs/%(program_name)s_%(process_num)s.log
+stderr_logfile = /var/praekelt/logs/%(program_name)s_%(process_num)s.log
+directory = /var/praekelt/unicore-mc
+user = ubuntu
+environment = DJANGO_SETTINGS_MODULE='project.settings'
+
+[program:unicore_mc_ws]
```

Comment:
@smn separate worker for websockets. I'm using `uwsgi` instead of `gunicorn` because I wasn't able to get it to run with `gunicorn` and it appears [it's not well supported](https://github.com/jrief/django-websocket-redis/issues/39)